src/providers/accident-provider.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
```

Comment:
Not sure we need this provider. We don’t use (and don’t plan to use) `setAccidentId` so there is no distinction between `initAccidentId` and `accidentId` in our use case. My preference would be to pass `accidentId` as prop – that would be just a couple of lines of code instead of 40+.

Reply:
We need this provider for the accident page anyway.